go/controller/internal/httpserver/handlers/modelconfig.go

```diff
@@ -169,14 +181,16 @@ func getStructJSONKeys(structType reflect.Type) []string {
 }
 
 type CreateModelConfigRequest struct {
-	Ref             string                      `json:"ref"`
-	Provider        Provider                    `json:"provider"`
-	Model           string                      `json:"model"`
-	APIKey          string                      `json:"apiKey"`
-	OpenAIParams    *v1alpha1.OpenAIConfig      `json:"openAI,omitempty"`
-	AnthropicParams *v1alpha1.AnthropicConfig   `json:"anthropic,omitempty"`
-	AzureParams     *v1alpha1.AzureOpenAIConfig `json:"azureOpenAI,omitempty"`
-	OllamaParams    *v1alpha1.OllamaConfig      `json:"ollama,omitempty"`
+	Name                  string                            `json:"name"`
```

Comment:
this is not correct -- the Ref has been added in the latest main and that's what should be used.